pkg/controller/configmap/configmap_controller.go

```diff
@@ -156,6 +195,7 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	} else if request.Namespace == "" && request.Name == operatortypes.NetworkCRDName {
 		reqLogger.Info("Reconciling cluster Network CRD change")
 	} else {
+		log.V(1).Info(fmt.Sprintf("Received change from unexpected namespace: %s", request.Namespace))
```

Comment:
how about setting log level to 4 or 6?

Reply:
Agreed. It's less important debug message.